[33mWARNING [0m urllib3.connectionpool:connectionpool.py:672 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002B9741C7E50>: Failed to establish a new connection: [WinError 10061] ç„¡æ³•é€£ç·šï¼Œå› ç‚ºç›®æ¨™é›»è…¦æ‹’çµ•é€£ç·šã€‚')': /wd/hub/session
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:672 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002B9741C7C40>: Failed to establish a new connection: [WinError 10061] ç„¡æ³•é€£ç·šï¼Œå› ç‚ºç›®æ¨™é›»è…¦æ‹’çµ•é€£ç·šã€‚')': /wd/hub/session
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:672 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002B9741C72E0>: Failed to establish a new connection: [WinError 10061] ç„¡æ³•é€£ç·šï¼Œå› ç‚ºç›®æ¨™é›»è…¦æ‹’çµ•é€£ç·šã€‚')': /wd/hub/session